Babel Macros - not working

Hi, it would seem I cannot import the macro version of styled-components:

https://www.styled-components.com/docs/tooling#babel-macro

    import styled from '@xstyled/styled-components/macro';

> Failed to compile.
> Error: Cannot find module '@xstyled/styled-components/macro'

Thought trying also importing from `@xstyled/macro` but no luck.

----

Importing directly from styled-components does work well:

    import styled from 'styled-components/macro';

----

### Background:

We have an app based on *create-react-app*, not *ejected*, and not using *rewire*, because *CRA* supports *babel macros*, and I wish to utilize that ability to be able to use the `babel-plugin-styled-components` plugin. 

I need this so my styled-components will be generated with a real display name, to be able to access components in *enzyme* unit tests.

[Related thread](https://github.com/styled-components/styled-components/issues/896#issuecomment-356756789)

--- a/website/pages/docs/getting-started/migrate-from-styled-components.mdx
+++ b/website/pages/docs/getting-started/migrate-from-styled-components.mdx
@@ -227,4 +227,19 @@ You can use it out of the box, but you have to specify xstyled as `topLevelImpor
 
 ## Babel Macro
 
-Unfortunately Babel Macro is not yet supported by xstyled.
+You may want to use [styled-components Babel macro](https://styled-components.com/docs/tooling#babel-macro) to use Babel plugin with zero-configuration.
+
+To use it, follow [styled-components guide](https://styled-components.com/docs/tooling#babel-macro) and add this configuration in your `package.json` or any other [babel-plugin-macros](https://github.com/kentcdodds/babel-plugin-macros) config file:
+
+```js
+// babel-plugin-macros.config.js
+module.exports = {
+  // ...
+  // Other macros config
+  styledComponents: {
+    importModuleName: '@xstyled/styled-components',
+  },
+}
+```
+
+You don't need to specify additional `topLevelImportPaths` config when you use babel macro.

